src/confidential.rs

```diff
@@ -182,6 +182,23 @@ impl Value {
             Value::Confidential(..) => 33,
         }
     }
+
+    /// Check if the value is explicit.
+    pub fn is_explicit(&self) -> bool {
+        match *self {
+            Value::Explicit(_) => true,
+            _ => false,
+        }
+    }
+
+    /// Unwrap the explicit value of this type.
+    /// Panics if [is_explicit] returns false.
```

Comment:
fn does not panic if it is not explicit it returns None

Reply:
ah damn missed that comment, thanks

fixed